xrmocap/transform/convention/keypoints_convention/__init__.py

```diff
@@ -110,7 +112,54 @@ def new_array_func(shape, value, ref_data, if_uint8):
         logger=keypoints.logger)
     return ret_kps
 
+def convert_bottom_up_kps_paf(
+    kps_paf: List,
+    src: str,
+    dst: str,
+    approximate: bool = False,
+    keypoints_factory: dict = KEYPOINTS_FACTORY,
+) -> Keypoints:
+    """Convert keypoints following the mapping correspondence between src and
+    dst keypoints definition.
+
+    Args:
+        keypoints (Keypoints):
+            An instance of Keypoints class.
+        dst (str):
+            The name of destination convention.
+        approximate (bool, optional):
+            Whether approximate mapping is allowed.
+            Defaults to False.
+        keypoints_factory (dict, optional):
+            A dict to store all the keypoint conventions.
+            Defaults to KEYPOINTS_FACTORY.
+
+    Returns:
+        Keypoints:
+            An instance of Keypoints class, whose convention is dst,
+            and dtype, device are same as input.
+    """
+
+    n_frame = len(kps_paf)
+    dst_n_kps = get_keypoint_num(
+        convention=dst, keypoints_factory=keypoints_factory)
+    dst_detections = []
+    for i in range(n_frame):
+        var = {'joints':[np.array([]) for j in range(dst_n_kps)], 'pafs':[np.array([]) for k in range(dst_n_kps)]}
+        dst_detections.append(var)
+    dst_idxs, src_idxs, _ = \
+        get_mapping(src, dst, approximate, keypoints_factory)
+    paf_mapping = all_paf_mapping[src][dst]
```

Comment:
Shall we get `paf_mapping` through `get_mapping`? If not, we'd better define the function name more specifically, for example, `openpose25_2_fourdag_paf_mapping`.

Reply:
we can't get `paf_mapping` from `get_mapping`, since it is obtained from limb convention.
`all_paf_mapping` is a class of `dict`, which defines paf mapping of multiple conventions.